ur_gazebo/launch/inc/ur_control.launch.xml

```diff
@@ -53,12 +53,16 @@
       "
     output="screen" respawn="false" />
 
-  <!-- Load and start the controllers listed in the 'controllers' arg. -->
-  <node name="ros_control_controller_spawner" pkg="controller_manager" type="spawner"
-    args="$(arg controllers)" output="screen" respawn="false" />
+  <group unless="$(eval arg('controllers') == '')">
```

Comment:
I don't think there's any reason for `group` if there's just one node in the group? Suggest reverting.

Reply:
I should actually have used if clause instead of this 😄